Bestseller.css

```diff
@@ -0,0 +1,555 @@
+        font-size: 10px;
+        cursor: pointer;    
+    }
+        
+    .linav:hover{
+        background-color: rgb(144,215,255);
+        border-radius: 5px;
+        font-size:15px ;
+    }
+        
+    .linav:active{
+        background-color:greenyellow;
+        border-radius: 5px; 
+    }
+    
+    #MQcont{
+        width: 100%;
+        height: 40px;
+        display: flex;
+        flex-direction: row;
+        justify-content:start;
+    }
+        
+    #buttonsearch{
+        width: 70%; 
+        height:30px;
+        margin-left: 4px;
+    }
+        
+    #searchbutton{
+        width: 100%;
+        height: 30px;
+        border:  solid 2px black;
+        border-radius: 5px;
+    } 
+        
+    #iconnavcont{
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 30%;
+        height:30px;
+        margin-left: 15px;
+        margin-right: 3px;
+    }
+        
+    .iconnav{
+        width: 20px;
+        height: 20px;
+        cursor: pointer;
+    }
+        
+    .iconnav:hover{
+        width: 5px;
+        height: 5px;
+    }
+    /* Media queries Best Seller  starts here */
+    #mainbestseller{
+    width: 100%;
+    height: 100%;
+    }
+    
+    #conttitlebs{
+    width: 100%;
+    height: 20%;
+    }
+    
+    #titlebs{
+    text-align: center;
+    font-size: 80px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    padding-top:20%;
+    }
+    
+    #span1{
+    color: rgb(241, 18, 167);
+    }
+    
+    #span2{
+    color:rgb(43, 150, 232);
+    }
+    
+    #contbooks{
+    width: 100%;
+    height: 79%;
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    }
+    
+    .contb{
+    width:30%;
+    display: flex;
+    flex-direction: column;     
+    align-items: center;
+    border: solid 2px lightgreen;
+    }
+    
+    .contimgbs{
+    width: 100%;
+    height: 80%;
+    display: flex;
+    align-items: center;
+    border-bottom:solid 2px lightblue;
+    }
+    
+    .imgbs{
+    width:50%;
+    height: 40%;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-left: 40px;
+    }
+    
+    .conticon{
+    width: 60%;
+    height: 30px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    }
+
+    .iconbs{
+    width: 20px;
+    height: 20px;
+    }
+    
+    .iconbs:hover{
+    width: 60px;
+    height: 60px;
+    }
+    
+    .conttextbs{
+    width: 80%;
+    height: 40px;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    }
+    
+    .namebs{
+    font-size: 12px;
+    font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    width: 100%;    
+    }
+    
+    .pricebs{
+    font-size: 13px;
+    font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    color: blue;
+    }
+    /* Media queries Footer starts here */
+    
+    #footer{
```

Comment:
Some clases has error with the indentation, please fix it, apart from that, everything looks great.